interface/client/templates/elements/nodeInfo.html

```diff
@@ -27,7 +27,8 @@
         {{else}}
             <i class="icon-feed"></i> {{TemplateVar.get "peerCount"}} {{i18n 'mist.nodeInfo.peers'}}
             &nbsp;&nbsp;|&nbsp;&nbsp;
-            <i class="icon-layers"></i> {{formattedBlockNumber}} <i class="icon-clock" style="margin-left: 10px;"></i> {{{timeSinceBlock}}}
+            <i class="icon-layers"></i> {{formattedBlockNumber}} 
+            <br> <i class="icon-clock" style="margin-left: 10px;"></i> {{{timeSinceBlock}}}
```

Comment:
This marginl-left might be removed i guess.. It looked a bit weird in the screen sharing when i saw it.